components/unit-converter.tsx

```diff
@@ -21,60 +21,73 @@ import {
   convertVolume,
   convertWeight,
 } from "@/lib/conversion-form-helper";
+import { VolumeTable, WeightTable, TemperatureTable } from "./conversion-table";
 
 const volumeUnits = Object.values(VolumeUnit);
 const weightUnits = Object.values(WeightUnit);
 const temperatureUnits = Object.values(TemperatureUnit);
 
 export default function UnitConverter() {
   const [input, setInput] = useState("");
+  const [conversionType, setConversionType] = useState(ConversionType.Volume);
```

Comment:
I think the need for this state value might not be needed if we move the tables inside the conversion form.Not sure how needed it would be but it would make the code cleaner.